packages/builder/src/pages/builder/portal/manage/users/_components/UpdateRolesModal.svelte

```diff
@@ -6,6 +6,8 @@
   export let app
   export let user
 
+  let devAccess = user?.admin?.global || user?.builder?.global
```

Comment:
I think this should apply to builder only, an admin user doesn't automatically get access in the portal and when trying to view the app directly

<img width="697" alt="Screenshot 2022-05-13 at 11 34 03" src="https://user-images.githubusercontent.com/8755148/168266201-87e140e4-7a2a-4c88-b266-e6720b3c95ca.png">
<img width="653" alt="Screenshot 2022-05-13 at 11 34 10" src="https://user-images.githubusercontent.com/8755148/168266209-e8d8082b-9fbb-4486-96e1-44edfde4d18a.png">

Reply:
Good catch - we can revoke dev access on the admin so that makes sense